src/components/sign-transactions-modal/sign-transactions-modal.types.ts

```diff
@@ -10,23 +10,26 @@ export type FungibleTransactionType = {
   imageURL: string;
 };
 
-export type TokenType = 'SemiFungibleESDT' | 'NonFungibleESDT' | 'FungibleESDT' | null;
+export type TokenType =
+  | 'MetaESDT'
+  | 'SemiFungibleESDT'
+  | 'NonFungibleESDT'
+  | 'FungibleESDT';
 
 export interface ISignTransactionsModalData {
   shouldClose?: true;
   commonData: {
     receiver?: string;
     data?: string;
     transactionsCount: number;
-    /**
-     * Token type of the transaction.
-     * @param {string} `null` - if is EGLD or MultiEsdt transaction.
-     */
     tokenType?: TokenType;
     egldLabel: string;
     feeLimit?: string;
     feeInFiatLimit?: string | null;
     currentIndex: number;
+    nextUnsignedTxIndex?: number;
```

Comment:
add TS Doc and add reference to the same file in sdk-dapp-core on the top of the file as a comment